tsdb/engine/tsm1/engine_test.go

```diff
@@ -1729,19 +1722,11 @@ type SeriesFile struct {
 
 // NewSeriesFile returns a new instance of SeriesFile with a temporary file path.
 func NewSeriesFile() *SeriesFile {
-	file, err := ioutil.TempFile("", "tsdb-series-file-")
+	dir, err := ioutil.TempDir("", "tsdb-series-file-")
```

Comment:
Do you need to change line `1743` (`SeriesFile.Close)` to use `RemoveAll` now that the series file is a collection of files?